data/json/clothing_mods.json

```diff
@@ -28,30 +161,143 @@
+      { "type": "encumbrance", "value": 1, "round_up": true, "proportion": [ "thickness" ] },
+      { "type": "storage", "value": 0.5, "round_up": false }
     ]
   },
   {
     "type": "clothing_mod",
-    "id": "furred",
-    "flag": "furred",
+    "id": "tailored_fur_collar",
+    "flag": "tailored_fur_collar",
     "item": "fur",
-    "implement_prompt": "Line it with fur",
-    "destroy_prompt": "Destroy fur lining",
+    "apply_flags": [ "COLLAR" ],
+    "exclude_flags": [ "SKINTIGHT" ],
+    "valid_parts": [ "TORSO" ],
+    "implement_prompt": "Add a standing fur collar",
+    "destroy_prompt": "Remove added collar",
+    "min_coverage": 80,
+    "difficulty": 3,
+    "mod_value": [ { "type": "warmth", "value": 5, "round_up": true }, { "type": "encumbrance", "value": 2, "round_up": true } ]
+  },
+  {
+    "type": "clothing_mod",
+    "id": "tailored_faux_fur_collar",
+    "flag": "tailored_faux_fur_collar",
+    "item": "faux_fur",
+    "copy-from": "tailored_fur_collar",
+    "implement_prompt": "Add a standing faux fur collar",
+    "destroy_prompt": "Remove added collar"
+  },
+  {
+    "type": "clothing_mod",
+    "id": "tailored_earplugs",
+    "flag": "tailored_earplugs",
+    "item": "ear_plugs",
+    "apply_flags": [ "PARTIAL_DEAF" ],
+    "exclude_flags": [ "DEAF" ],
+    "valid_parts": [ "HEAD" ],
+    "implement_prompt": "Add makeshift hearing protection",
+    "destroy_prompt": "Remove hearing protection",
+    "min_coverage": 60,
+    "no_material_scaling": true,
+    "item_quantity": 1,
+    "difficulty": 1,
+    "mod_value": [ { "type": "encumbrance", "value": 3, "round_up": true } ]
+  },
+  {
+    "type": "clothing_mod",
+    "id": "tailored_long_sleeves",
+    "flag": "tailored_long_sleeves",
+    "item": "rag",
+    "valid_parts": [ "ARMS" ],
+    "exclude_flags": [ "BELTED", "tailored_short_sleeves" ],
+    "implement_prompt": "Lengthen the sleeves",
+    "destroy_prompt": "Restore the sleeves",
+    "min_coverage": 50,
+    "max_coverage": 70,
+    "difficulty": 1,
+    "no_material_scaling": true,
+    "item_quantity": 12,
+    "mod_value": [ { "type": "encumbrance", "value": 3, "round_up": true }, { "type": "warmth", "value": 10, "round_up": true } ]
+  },
+  {
+    "type": "clothing_mod",
+    "id": "tailored_short_sleeves",
+    "flag": "tailored_short_sleeves",
+    "item": "rag",
+    "valid_parts": [ "ARMS" ],
+    "exclude_flags": [ "BELTED", "tailored_long_sleeves" ],
+    "implement_prompt": "Shorten the sleeves",
+    "destroy_prompt": "Restore the sleeves",
+    "min_coverage": 70,
+    "max_coverage": 100,
+    "item_quantity": 0,
+    "no_material_scaling": true,
+    "mod_value": [ { "type": "encumbrance", "value": -2, "round_up": false }, { "type": "warmth", "value": -5, "round_up": false } ]
+  },
+  {
+    "type": "clothing_mod",
+    "id": "tailored_fingerless",
+    "flag": "tailored_fingerless",
```

Comment:
Should have applied flag of ALLOWS_NATURAL_ATTACKS, as the crafted fingerless gloves do.

Reply:
Nice thank you.